src/Writing/SVGWriter.php

```diff
@@ -50,23 +50,30 @@ public function writeNode(SVGNode $node)
         $this->appendAttributes($node->getSerializableAttributes());
         $this->appendStyles($node->getSerializableStyles());
 
-        if (!($node instanceof SVGNodeContainer) && !($node instanceof SVGStyle)) {
-            $this->outString .= ' />';
+        $textContent = htmlspecialchars($node->getValue());
```

Comment:
@meyfa @micgro42  Why the value is escaped? It prevent to inject complete svg into a node, may the escape can be removed?

Reply:
See the commit where this was actually introduced: c2b80145bbde49ea21a44faa546de961c47dd8fe

The value of a node is intended to be pure text. If you want to add more SVG nodes, then add them as children of a container node.